contrib/cirrus/integration_test.sh

```diff
@@ -14,7 +14,8 @@ req_env_var "
 dnf install -y make glib2-devel git gcc golang
 setup_gopath
 cd $CIRRUS_WORKING_DIR
+go mod init || true
```

Comment:
For the benefit of some future maintainer, would you mind dropping a short a comment about why failure can be ignored?